src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/CrmServiceAdapter.cs

```diff
@@ -89,12 +115,18 @@ public static void ImportWordTemplate(this CrmServiceClient svcClient, string fi
 
             if (documentTemplate.Id == Guid.Empty)
             {
-                svcClient.Create(documentTemplate);
+                crmSvc.Create(documentTemplate);
             }
             else
             {
-                svcClient.Update(documentTemplate);
+                crmSvc.Update(documentTemplate);
             }
         }
+
+        public IOrganizationService GetOrganizationService() => (IOrganizationService)crmSvc.OrganizationWebProxyClient ?? crmSvc.OrganizationServiceProxy;
+
+        public EntityCollection RetrieveMultiple(QueryByAttribute query) => crmSvc.RetrieveMultiple(query);
+
+        public void Update(Entity record) => crmSvc.Update(record);
```

Comment:
Can `CrmServiceClient` be inherited or is it `sealed`? That would save having to write wrappers for the methods.

Reply:
It is sealed 😞